File: C++/catch_exception.cpp
#include<iostream>
using namespace std;
int main()
{
    int a,b,x;
    cout<<"Enter the value of A : ";
    cin>>a;
    cout<<"Enter the value of B : ";
    cin>>b;
    x=a/b;
    try
    {
        if(x!=0)
        cout<<"\n Result is"<<x;
        else
        throw x;
    }
    catch (int m)
    {
        cout<<"\n Integer exception caught";
    }
    return 0;
}

File: Java/Catch_Exception.java
import java.util.Scanner;

class Catch_Exception {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a, b, res;
        System.out.println("Enter Value of A : ");
        a = sc.nextInt();
        System.out.println("Enter Value of B : ");
        b = sc.nextInt();
        try {
            res = a / b;
            System.out.println("Result = " + res);
        } catch (Exception e) {
            System.out.println("Exception Caught : " + e);
            
        }
        sc.close();
    }
}
